source/module_elecstate/module_dm/test/test_cal_dm_R.cpp

```diff
@@ -4,6 +4,51 @@
 #include "gtest/gtest.h"
 #include "module_elecstate/module_dm/density_matrix.h"
 #include "module_hamilt_lcao/module_hcontainer/hcontainer.h"
+#include "module_cell/klist.h"
+
+K_Vectors::K_Vectors()
+{
+}
+
+K_Vectors::~K_Vectors()
+{
+}
+int K_Vectors::get_nks() const
```

Comment:
Could you directly define get_nks() and related functions in klist.h?

Reply:
Do you mean that we define the function directly in klist.h instead of klist.cpp?